src/OrchardCore.Modules/OrchardCore.Menu/Migrations.cs

```diff
@@ -14,10 +14,16 @@ public Migrations(IRecipeMigrator recipeMigrator)
         }
 
         public async Task<int> CreateAsync()
+        {
+            // Return 1 to shortcut the second migration on new content definition schemas.
```

Comment:
So what you need to do here is have run the main migration 

`await _recipeMigrator.ExecuteAsync("menu.recipe.json", this);`